R/connect.R

```diff
@@ -62,11 +63,11 @@ setMethod("dbConnect", "MySQLDriver",
   function(drv, dbname = NULL, username = NULL, password = NULL, host = NULL,
     unix.socket = NULL, port = 0, client.flag = 0,
     groups = "rs-dbi", default.file = NULL, ssl.key = NULL, ssl.cert = NULL,
-    ssl.ca = NULL, ssl.capath = NULL, ssl.cipher = NULL, ...) {
+    ssl.ca = NULL, ssl.capath = NULL, ssl.cipher = NULL, timeout=3600, ...) {
```

Comment:
In any case, if you find any other default value is better, I have no problem with that.